src/Aspire.Dashboard/Components/Controls/StructuredLogDetails.razor

```diff
@@ -18,6 +19,7 @@
                       ValueColumnValue="(vm) => vm.Value"
                       NameSort="_nameSort"
                       ValueSort="_valueSort"
-                      HighlightText="@_filter" />
+                      HighlightText="@_filter"
+                      WrappedProperties="@ImmutableArray.Create("ex.Message")"/>
```

Comment:
What is the special meaning of `ex.Message`?

These are the standard OTEL properties:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/ebb35248003885c6a3315417d090fb081fccd2b0/src/Shared/SemanticConventions.cs#L96-L100

Can you investigate why you have an exception with non-standard properties.

Reply:
I don't think hardcoding known properties is a good long term situation here. There are many tools that display structured logs out there. What do they do in their UI?

I feel like there should be a way to select a property row and have it switch to multiline, but I don't have another tool that does that off the top of my head as prior art.

Reply:
> I feel like there should be a way to select a property row and have it switch to multiline text box that preserves whitespace

I have not checked others, but this does not exist in the msbuild log viewer; it displays as one line.

The solution for now should then just be to wrap all properties. No need to add a toggle

Reply:
msbuild log viewer is probably made for devs, by devs. We should look at what other tools in this space do. What does AppInsights in azure do? Or datadog, seq, etc.

Reply:
There are others that we will need to be expandable. Can it be based on the content truncation?

Reply:
> Can it be based on the content truncation

possibly, but I don't know how to do this